### PR TITLE
Daofund TokenFallback Update

### PR DESCRIPTION
**Pull request by Suyog007**
new enumerable setdb added
taking address as a dictdb key

### CODE REVIEW

File: core_contracts/daofund/daofund.py [hunk 1]
```diff
@@ -101,6 +104,15 @@ def addSymbolToSetdb(self) -> None:
         for symbol in assets:
             self._symbol.add(symbol)
 
+    @external
+    @only_owner
+    def addAddressToSetdb(self) -> None:
+        for symbol in self._symbol.range(0, len(self._address)):
```
Comment (dbrehmer) [inline core_contracts/daofund/daofund.py]: Doesn't this need to be len(self._symbol)? self._address will start at len = 0.

File: core_contracts/daofund/daofund.py [hunk 2]
```diff
@@ -199,13 +211,11 @@ def tokenFallback(self, _from: Address, _value: int, _data: bytes) -> None:
         address = str(self.msg.sender)
         for symbol in assets:
```
Comment (dbrehmer) [inline core_contracts/daofund/daofund.py]: Seems like we do not really need to reference Loans at all, right? Seems line only lines 211, 216, 217, and 218 are needed.

**Review by dbrehmer — CHANGES_REQUESTED** [review]
Needs a couple changes.

File: core_contracts/daofund/daofund.py (hunk 1 above)
Comment (Suyog007) [reply core_contracts/daofund/daofund.py]: yes

**Review by dbrehmer — APPROVED** [review]
Looks good.

**Pull request merged**